nidmviewerfsl/lib/pageGen.py

```diff
@@ -450,20 +450,35 @@ def generatePostStatsHTML(graph, postStatsFilePath="postStats.html"):
 
             # Add the image. If we have FSL the image was found in the pack.
             if askFSL:
-                postStats += img(
-                    src='data:image/jpg;base64,' +
-                    encodeImage(
-                        os.path.join(
-                            os.path.split(
-                                postStatsFilePath)[0],
-                            excursionSetSliceImage[i])
-                        ).decode())
+
+                # If the slice image already exists add it.
```

Comment:
Suggestion of edit to make the code more generic: 
 1. Always check whether the slice image is present (regardless of software) and if absent recreate the slice view.
 2. Use a variable to store the software name and then use the same code for all software if the slice view is missing

?